### PR TITLE
use paket3 to streamline build process

### PR DESCRIPTION
**Pull request by cloudRoutine**

### CODE REVIEW

File: build.cmd [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @echo off
 cls
 
-.paket\paket.bootstrapper.exe
+.paket\paket.bootstrapper.exe 3.0.0-alpha040
```
Comment (Krzysztof-Cieslak) [inline build.cmd]: Maybe use `prerelease` instead of giving hard-coded version?

**Comment by Krzysztof-Cieslak** [comment]
Things in `paket-files` are tracked, you've gitignored them later but since in earlier commit there were added, you have to remove them.

**Comment by Krzysztof-Cieslak** [comment]
Also maybe try using this convention - https://github.com/fsprojects/Paket/pull/1353#issuecomment-169735534 - for adding output dlls from Helpers.

**Comment by Krzysztof-Cieslak** [comment]
Or we can create local nupkg file in Helpers build process just for sake of easier adding it to other projects. http://fsprojects.github.io/Paket/git-dependencies.html#Using-Git-repositories-as-NuGet-source

File: build.cmd (hunk 1 above)
Comment (cloudRoutine) [inline build.cmd]: can't at the moment https://github.com/fsprojects/Paket/issues/1464 :(
easy fix later

**Comment by cloudRoutine** [comment]
I don't think the http convention makes it any easier you still have to reference the paths inside of `paket-files` so I'm not sure what the benefit is there.
I thought the whole point of using the `git` dependency was so we didn't have to create a nuget package, if we're making a nuget package what's the benefit of using a git dependency anyway? Plus even with a nuget package we'll still have to point into `packages` instead of `paket-files` and reference all the dlls individually, so it just adds some more boilerplate on the `ionide-helpers` side and another line or two in this repo without making anything simpler, so I don't see the benefit of that either.
Unless I'm missing something?

**Pull request merged**